src/it/scala/io/iohk/ethereum/sync/util/CommonFakePeer.scala

```diff
@@ -298,21 +300,88 @@ abstract class CommonFakePeer(peerName: String, fakePeerCustomConfig: FakePeerCu
     (newBlock, newWeight, parentWorld)
   }
 
+  private def generateInvalidBlock(
```

Comment:
Perhaps put a comment where / how this block is made invalid?

Also, below is a method that mentions "MaliciousBlock", is this the same thing? If so, would be better to keep a consistent name.